weaver/core/network/corda-interop-app/makefile

```diff
@@ -27,13 +27,8 @@ publish-contracts: github.properties
 test:
 	./gradlew test
 
-.PHONY: clean-local
-clean-local:
-	./gradlew clean
-	rm -rf fabric-protos proto || true
-	rm -rf .gradle
-
```

Comment:
Why remove this?

Reply:
both `clean-local` and `clean` were doing same thing, so keeping only one `clean` in makefile

Reply:
Also one point, since `protos` are now built into `common/protos-java-kt`, these folders `fabric-protos` and `protos` are never created in this directory. (should have changed this long ago, but missed it)